resources/function_help/json/overlay_nearest

```diff
@@ -2,55 +2,64 @@
   "name": "overlay_nearest",
   "type": "function",
   "groups": ["GeometryGroup"],
-  "description": "This returns an array of results of an expression evaluated on features from a different layer ordered BY DISTANCE to the current feature, or, if no expression if provided, simply returns whether at least one feature from the other layer was found. Note : this function can be slow and consume a lot of memory for large layers.",
+  "description": "Returns whether the current feature has feature(s) from a target layer within a given distance, or an array of expression-based results for the features in the target layer within a distance from the current feature.<br><br>Note: This function can be slow and consume a lot of memory for large layers.",
   "arguments": [
     {
       "arg": "layer",
-      "description": "the other layer"
+      "description": "the target layer"
     },
     {
       "arg": "expression",
-      "description": "an optional expression to evaluate on the features from the other layer (if not set, the function will just return a boolean indicating whether there is at least one match)",
+      "description": "an optional expression to evaluate on the features from the target layer. If not set, the function will just return a boolean indicating whether there is at least one match.",
       "optional": true
     },
     {
       "arg": "filter",
-      "description": "an optional expression to filter the matching features (if not set, all features will be returned)",
+      "description": "an optional expression to filter the target features to check. If not set, all the features in the target layer will be used.",
       "optional": true
     },
     {
       "arg": "limit",
-      "description": "an optional integer to limit the number of matching features (if not set, only the nearest feature will be returned)",
-      "optional": true
+      "description": "an optional integer to limit the number of matching features. If not set, only the nearest feature will be returned. If set to -1, returns all the matching features.",
+      "optional": true,
+      "default": "1"
```

Comment:
I notice that limit:=0 has the same result as 1, and any value <0 returns all the matching features.
Not sure if of any relevance for users but wanted to share...